docs/development.rst

```diff
@@ -282,6 +282,103 @@ For further discussion of good and bad practices regarding commits, see:
 .. _How to Get Your Change Into the Linux Kernel: https://www.kernel.org/doc/Documentation/SubmittingPatches
 
 
+.. _`Making a change`:
+
+Making a change
+---------------
+
```

Comment:
I made a couple of change proposals for pywbem in this area.  They apply here, just to clarify a bith this whole process.

Reply:
DONE